Code/Query/RangeQuery.h

```diff
@@ -93,6 +90,7 @@ class RangeQuery
     res->setEndsOpen(this->df_lowerOpen, this->df_upperOpen);
     res->setDataFunc(this->d_dataFunc);
     res->d_description = this->d_description;
```

Comment:
Just a comment, it seems like these should be in the base class as this code appears multiple time.  This seems like an easy thing to miss.